source/gui/blockAction.py

```diff
@@ -1,25 +1,42 @@
 # A part of NonVisual Desktop Access (NVDA)
-# Copyright (C) 2023 NV Access Limited, Cyrille Bougot
+# Copyright (C) 2023-2024 NV Access Limited, Cyrille Bougot
 # This file is covered by the GNU General Public License.
 # See the file COPYING for more details.
 
+from collections.abc import Callable
```

Comment:
why was this changed from the `typing` import?

Reply:
Since Python 3.9, [`typing.Callable` is a deprecated alias to `collections.abc.Callable`](https://docs.python.org/3/library/typing.html#typing.Callable).